src/python/pants/core/util_rules/environments.py

```diff
@@ -201,6 +201,15 @@ class DockerImageField(StringField):
         """
     )
 
+class DockerBindMountsField(StringSequenceField):
+    alias = "bind_mounts"
+    help = help_text(
+        """
+        Read-only bind mounts to use when starting the container.
+
+        The values should be in the form "<path on host>:<path on container>"
```

Comment:
The user could potentially also specify a Docker volume to mount as well. Maybe just link the relevant Docker documentation?